pages/landing/patcher/index.html

```diff
@@ -101,27 +101,27 @@
 features:
   - title: Update dependency versions
     description: Like DependaBot, Patcher automatically updates you to the latest versions of dependencies
-    icon: /assets/img/landing-zone-features/features-7@3x.png
+    icon: '<i class="fa fa-refresh fa-4x" aria-hidden="true"></i>'
```

Comment:
```suggestion
    icon: '<i class="fa fa-refresh fa-3x" aria-hidden="true"></i>'
```

NIT: total matter of preference, but I _think_ the icons look a tiny bit better one size smaller.